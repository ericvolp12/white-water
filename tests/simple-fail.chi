start -n node-1
start -n node-2
start -n node-3
start -n node-4

wait -t 2

set -n node-1 -k A -v 1
set -n node-1 -k B -v 2
set -n node-1 -k C -v 3
set -n node-1 -k D -v 4

wait -t 2

fail_node -n node-1

wait -t 5

get -n node-2 -k A
get -n node-2 -k B
get -n node-2 -k C
get -n node-2 -k D
set -n node-2 -k E -v 5

recover_node -n node-1

wait -t 5

fail_node -n node-2

wait -t 5

get -n node-1 -k E