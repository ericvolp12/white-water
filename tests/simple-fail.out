SET id=1 OK: A = 1
SET id=2 OK: B = 2
SET id=3 OK: C = 3
SET id=4 OK: D = 4
GET id=5 OK: A = 1
GET id=6 OK: B = 2
GET id=7 OK: C = 3
GET id=8 OK: D = 4
SET id=9 OK: E = 5
ERROR: GET id=10 failed (k=E): E= 5 : Current Leader is

// This last line is failing because it isn't the leader but its reported value is 5, so we're good.